apps/site/assets/css/_amenity.scss

```diff
@@ -15,3 +15,49 @@
     margin: .65rem;
   }
 }
+
+.facilities-list-header {
+  background: $brand-primary-darkest;
+  color: $white;
+  height: 3.75rem;
+  padding-top: 0 !important;
```

Comment:
Question: What is the `!important` overriding?

Reply:
It's overriding the `padding-top` in `elevator-row` for the header bar